dist/index.js

```diff
@@ -6,18 +6,18 @@ var __getProtoOf = Object.getPrototypeOf;
 var __hasOwnProp = Object.prototype.hasOwnProperty;
 var __markAsModule = (target) => __defProp(target, "__esModule", { value: true });
 var __commonJS = (cb, mod) => function __require() {
-  return mod || (0, cb[Object.keys(cb)[0]])((mod = { exports: {} }).exports, mod), mod.exports;
```

Comment:
Those changes are due to the `prepare` script which runs `esbuild`, and `esbuild` having been updated as part of some other PR.

Reply:
Asking questions but why is the `dist` output checked into git?